File: Implementation/python/07_주사위세개.py
# fast campus 강의
# https://www.acmicpc.net/problem/2480
# 1

arr = list(map(int, input().split()))

n = len(set(arr))

if n == 1:
    print(10000 + arr[0] * 1000)
elif n == 2:
    arr.sort()
    print(1000 + arr[1] * 100)
else:
    print(max(arr) * 100)

File: Implementation/c++/07_주사위세개.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/2480
// 1

#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
	vector<int> dice(3);
	for (int i = 0; i < 3; i++)
		cin >> dice[i];
	sort(dice.begin(), dice.end());

	if (dice[0] == dice[2])
		cout << 10000 + dice[0] * 1000;
	else if ((dice[0] == dice[1]) || (dice[1] == dice[2]))
		cout << 1000 + dice[1] * 100;
	else
		cout << dice[2] * 100;
}